finding/finding_test.go

```diff
@@ -198,3 +199,88 @@ func Test_FromBytes(t *testing.T) {
 		})
 	}
 }
+
+func TestOutcome_UnmarshalYAML(t *testing.T) {
+	type args struct {
+		n *yaml.Node
+	}
+	tests := []struct { //nolint:govet
+		name    string
+		o       Outcome
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "positive outcome",
+			o:    OutcomePositive,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "Positive",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "negative outcome",
+			o:    OutcomeNegative,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "Negative",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "NotAvailable outcome",
+			o:    OutcomeNotAvailable,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "NotAvailable",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "NotSupported outcome",
+			o:    OutcomeNotSupported,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "NotSupported",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "Unknown error",
+			o:    OutcomeError,
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "Error",
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name: "Unknown outcome",
+			args: args{
+				n: &yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: "Unknown",
+				},
+			},
+			wantErr: true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if err := tt.o.UnmarshalYAML(tt.args.n); (err != nil) != tt.wantErr {
```

Comment:
we don't want to unmarshall until the value we're looking for.
 
I think you need a local var here to unmarshall into, and then a comparison between that value and what we wanted?

right now we're only testing for an error.

Reply:
Agreed! Fixed it